Data/ContentStore.hs

```diff
@@ -133,11 +172,49 @@ doStore cs hasher writer object = do
 
     -- Only store the object if it does not already exist in the content store.
     -- If it's already there, just return the digest.
-    unlessM (liftIO $ doesFileExist path) $
-        liftIO $ writer path object
+    unlessM (liftIO $ doesFileExist path) $ liftIO $ do
```

Comment:
So much liftIO... I think this could be rewritte as:
```liftIO $ unlessM (doesFileExist path) $ do```

Reply:
Done